swarm/tools/flow_studio_ui/fragments/10-header.html

```
@@ -18,8 +18,8 @@ <h1>Flow Studio</h1>
         <div id="tour-menu" class="tour-dropdown-menu" role="menu" aria-label="Available tours" data-uiid="flow_studio.header.tour.menu"></div>
       </div>
       <div class="mode-toggle" role="group" aria-label="View mode" data-uiid="flow_studio.header.mode">
-        <button id="mode-author" class="active" title="Show all paths, CLI commands, and technical details" aria-pressed="true" data-uiid="flow_studio.header.mode.author">Author</button>
-        <button id="mode-operator" title="Simplified view focused on run status and decisions" aria-pressed="false" data-uiid="flow_studio.header.mode.operator">Operator</button>
+        <button id="mode-author" class="active" title="Show all paths, CLI commands, and technical details" aria-label="Toggle author mode" aria-pressed="true" data-uiid="flow_studio.header.mode.author">Author</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)